### PR TITLE
[CRDOM203][5R] add DocType rule as ecorule

### PR DESCRIPTION
**Pull request by Silicoman**
As discuss here, https://github.com/green-code-initiative/ecoCode/issues/103

Add new rule to label as eco-design

### CODE REVIEW

File: tools/rules_config/_config.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ TAG_ECOCONCEPTION=test1
 # TAG_ECOCONCEPTION=
 
 # list of rule keys that will be updated with new tag
-RULES_KEYS=css:S4655,php:S2014,Web:ItemTagNotWithinContainerTagCheck
+RULES_KEYS=css:S4655,php:S2014,Web:ItemTagNotWithinContainerTagCheck,Web:DoctypePresenceCheck
```
Comment (dedece35) [inline tools/rules_config/_config.sh:29]: please remove other rules because they are only for testing the system

**Review by dedece35 — CHANGES_REQUESTED** [review]
before merging this PR, waiting for #17, thus I modify this PR to Draft.
Then, add an explanation of adding this rule as "eco-design" type

**Review by dedece35 — DISMISSED** [review]

**Comment by Silicoman** [comment]
i Have resolved with the new script. It's ok, logs below.
I have deleted examples from main.

I'm waiting infos from [issue opened if it's a relevant. ](https://github.com/green-code-initiative/ecoCode/issues/103)

```
--- DEBUG --- Read file ./SONAR_RULES_REUSED.md
--- DEBUG --- Extracting rules: Web:DoctypePresenceCheck

--- DEBUG --- SONAR_TOKEN : xxxxxxxxxxxxxxxxxxxxxxxxx
--- DEBUG --- SONAR_URL : http://localhost:9000
--- DEBUG --- RULES_KEYS : Web:DoctypePresenceCheck
--- DEBUG --- TAG_ECOCONCEPTION : test1

*****  Checking SonarQube connection  *****
--- DEBUG --- SonarQube connection ok : 200

*****  Processing rule Web:DoctypePresenceCheck  *****
Existing sysTags : [
  "user-experience"
]
Existing tags : []
--- DEBUG ---   Processing systag user-experience
  Tag test1 NOT FOUND in tags array nor in systags array : we will add it to rule 'Web:DoctypePresenceCheck'
Execution of command : curl -u xxxxxxxxxxxxxxxxxxxxxxx: -s -o /dev/null --request POST "http://localhost:9000/api/rules/update?key=Web:DoctypePresenceCheck&tags=test1"
CALL to Sonar API to add tag

==> 1 rules updated
```

**Comment by dedece35** [comment]
Hi @Silicoman,
ok for me, now.
did you think about our documentation on how to upgrade our installation doc (for developers and clients) ?

**Comment by dedece35** [comment]
Hi @Silicoman,
after checking, I don't find rule `Web:DoctypePresenceCheck` in SonarQube :(
thus, modification isn't possible.

Do you have this rule when you install SonarQube 10, locally ?

**Comment by dedece35** [comment]
Hi @Silicoman and @utarwyn 
any news for this rule ?
do you know this rule id ?
if not, I will refuse this PR.

**Comment by utarwyn** [comment]
Hello @dedece35,
based on our discussion in https://github.com/green-code-initiative/ecoCode/issues/103, this rule doesn't seem to be eco-design as it stands. So I propose to also close this PR because the `eco-design` tag should not be added to the SonarQube rule.

**Comment by dedece35** [comment]
Hi @Silicoman @utarwyn 
like discussed in previous PR, I close this PR without merging it.

**Pull request closed without merging**